### ISSUE TITLE
[REQ] Include background downloading Roblox updates before the official launcher does

### ISSUE THREAD

**Issue by exurd** [bloxstraplabs/bloxstrap#2845]
### Acknowledgement of preliminary instructions

- [X] I have read and acknowledged the preliminary instructions.

### What idea do you have?

In RDC 2024, Roblox announced that background downloads would be included in the launcher in the near future. Let's get ahead of the curve and add this feature to Bloxstrap before then.

**Comment by pizzaboxer** [comment]
No. It is a nice-to-have, but it isn't necessary, and I don't want to be dealing with extra complications on top of all the work on the bootstrapper/updater that I'm currently having to do already.

**Comment by pizzaboxer** [comment]
Actually, implementing it might not be as bad as I first thought.

**Comment by exurd** [comment]
Closed due to background updates being added in tag [v2.9.0](https://github.com/bloxstraplabs/bloxstrap/releases/tag/v2.9.0), PR #4861, commit ca36306.
